CHANGELOG.md

```diff
@@ -1,5 +1,69 @@
 # Smithy Typescript Codegen Changelog
 
+## 0.13.0 (2023-01-25)
+
+### Features
+
+* feat(smithy-typescript-codegen): allow deferred resolution for api key config ([#588](https://github.com/awslabs/smithy-typescript/pull/588))
```

Comment:
remove `feat`, `chore`, `fix` from here. The entries here may need to be edited from the PR titles.